README.md

```diff
@@ -62,5 +62,116 @@ EOF
 ```
 See [WEBHOOKS.md](docs/WEBHOOKS.md) for instructions on how to enable webhooks.
 
+### Customizing provider deployment
+
+Crossplane uses `DeploymentRuntimeConfig` object to apply customizations on the provider.
+Here are a few examples:
+
+
+```yaml
+apiVersion: pkg.crossplane.io/v1beta1
+kind: DeploymentRuntimeConfig
+metadata:
+  name: provider-ceph
+spec:
+  deploymentTemplate:
+    spec:
+      selector: {}
+      template:
+        spec:
+          containers:
+          - name: package-runtime
+            args:
+            - --kube-client-rate=80000
+            - --reconcile-timeout=5s
+            - --max-reconcile-rate=600
+            - --reconcile-concurrency=160
+            - --poll=30m
+            - --sync=1h
+            - --assume-role-arn=[ASSUME_ROLE_ARN]
+```
+
+You have to attach `DeploymentRuntimeConfig` to the `Provider` object.
+
+```yaml
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-ceph
+spec:
+  runtimeConfigRef:
+    name: provider-ceph
+```
+
+### Enabling validation webhook
```

Comment:
I think we need to make it clear that this is for your development/debugging cluster. 

How about we create a file `docs/DEVELOPMENT.md` which contains the above info on spinning up a local cluster for dev _and_ this section on enabling the webhook locally?

If you want to just merge this PR and we can do it in a follow-up then I'm fine with that too.

Reply:
@nolancon This section is for production too. This is the way how customers have to enable webhook.

Reply:
@nolancon you are right, here is the update docs: https://github.com/linode/provider-ceph/pull/172